Harbour/UI/Views/ContainerDetailsView/ContainerDetailsView.swift

```diff
@@ -269,6 +269,11 @@ struct ContainerDetailsView: View {
 				}
 				.disabled(viewModel.containerDetails?.mounts == nil)
 
+				NavigationLink(value: Subdestination.devices) {
+					Label("ContainerDetailsView.Section.Devices", systemImage: SFSymbol.terminal)
+				}
+				.disabled(viewModel.containerDetails?.hostConfig.devices == nil)
```

Comment:
Devices aren't used that often, please hide it by default. Also swap the icon to `externaldrive`.

```suggestion
				if !(viewModel.containerDetails?.hostConfig.devices?.isEmpty ?? true) {
					NavigationLink(value: Subdestination.devices) {
						Label("ContainerDetailsView.Section.Devices", systemImage: "externaldrive")
					}
				}
```